web.beans/src/org/netbeans/modules/web/beans/resources/Bundle.properties

```diff
@@ -14,6 +14,7 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+
```

Comment:
This is not necessary.

Reply:
Just trying to be coherent - some file has other not. What do you suggest?

Reply:
It is not related to relicensing. So you should not change this file.